code/obj/machinery/computer/syndicate/podbay.dm

```diff
@@ -0,0 +1,123 @@
+// Riot computer but reflavored for the Cairngorm
+
+/obj/machinery/computer/battlecruiser_podbay
+	name = "Podbay Authorization"
+	icon_state = "drawbr"
+	density = 0
+	glow_in_dark_screen = TRUE
+	/// How many authentications are needed to release the shielding on the podbay. Set the first time someone clicks this.
+	var/auth_need = null
+	var/list/authorized
+	desc = "Controls access to the podbay. Use this when your team is ready to go."
+
+	light_r = 1
+	light_g = 0.3
+	light_b = 0.3
+
+	/// Whether or not the podbay on the Cairngorm is authorized.
+	var/authed = FALSE
+	/// How long until we take matters into our own hands.
+	var/auth_delay = 10 MINUTES
+	var/area/battlecruiser_area
+
+	New()
+		..()
+		START_TRACKING
+
+	initialize()
+		battlecruiser_area = get_area_by_type(/area/syndicate_station/battlecruiser)
+		SPAWN(auth_delay)
+			authorize() // If they haven't done it before auth_delay, do it for em
+		..()
+
+	disposing()
+		STOP_TRACKING
+		..()
+
+	proc/authorize()
+		if(src.authed)
+			return
+
+		logTheThing(LOG_STATION, usr, "authorized Cairngorm podbay access")
+		authed = 1
+		src.ClearSpecificOverlays("screen_image")
+		src.icon_state = "drawbr-alert"
+		src.UpdateIcon()
+
+		for (var/obj/forcefield/battlecruiser/S in battlecruiser_area)
```

Comment:
any `in area` check is actually a secret `in world` check, I'd suggest to track the forcefields and delete them that way instead of this

Reply:
I have no idea how to track the forcefields so I asked in imcoder

Reply:
ok done